build/ci/evergreen.yml

```diff
@@ -517,6 +517,44 @@ tasks:
           MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
           MCLI_SERVICE: cloud
           E2E_TAGS: atlas,clusters,m0
+  - name: atlas_interactive_setup_e2e
```

Comment:
I would suggest keeping all atlas interactive tests in one variant `atlas_interactive_e2e` this way more tests would run at once

Reply:
Currently, we have 2 tasks:
One for setup and one for quickstart and both are under the same variant `e2e_iteractive`, is this your suggestion? Or do you mean all of them (quickstart and setup) under the same task as well? If the latter, I can update right away, just wanted to clarify. Thanks!

Reply:
the full story here is if your test is too fast (under 7 min) you will be spending more time waiting for a host than running tests, better to keep slightly bigger test variants

Reply:
cool! so the second option if I understood correctly. thanks for the extra info